ser/infer.py

```diff
@@ -0,0 +1,36 @@
+import torch
+
+@torch.no_grad()
+def test_model_inference(params, model, image, label):
+    print(f"Runing inference on model - {params.name}")
+    print(f"Hyperparameters:\n",
+          f"EPOCHS: {params.epochs}\n",
+          f"LEARNING RATE: {params.learning_rate}\n",
+          f"Label of image: {label}"
+        )
+    model.eval()
+    output = model(image)
+    prediction = torch.argmax(output, 1, keepdim=True).item()
+    confidence = max(list(torch.exp(output).numpy()[0]))
+    
+    pixels = image[0][0]
+    print(generate_ascii_art(pixels))
+
+def generate_ascii_art(pixels):
+    ascii_art = []
```

Comment:
As this is not explicitly an inference related function, you could put it in another file (e.g a image_utils.py or something)